react/Viewer/MobileActions.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import useBreakpoints from '../hooks/useBreakpoints'
+
+import styles from './styles.styl'
+
+const MobileActions = ({ children }) => {
+  const { isMobile } = useBreakpoints()
+
+  if (!isMobile) return null
+  return <div className={styles['viewer-mobileActions']}>{children}</div>
+}
+
+export default MobileActions
```

Comment:
I dont' think this logic should be in UI. The app should be responsible for this check. no?